### ISSUE TITLE
firebase performance monitoring

### ISSUE THREAD

**Issue by doczoidberg** [angular/angularfire2#2068]
Is there a plan to integrate the new [firebase performance monitoring](https://firebase.google.com/docs/perf-mon/get-started-web) in angularfire?

**Comment by jamesdaniels** [comment]
#2064 `npm i --save @angular/fire@next`

**Comment by stot3** [comment]
Right now using firebase@6.0.x is broken because of the firebase perfomance monitoring. 

ERROR in node_modules/@angular/fire/firebase.app.module.d.ts(17,22): error TS2420: Class 'FirebaseApp' incorrectly implements interface 'App'.
  Property 'performance' is missing in type 'FirebaseApp'.

**Comment by jamesdaniels** [comment]
@stot3 this was addressed in `5.1.3`

**Comment by bogacg** [comment]
I'm getting this error:
```
ERROR in ./node_modules/@angular/fire/performance/es2015/performance.module.js
Module not found: Error: Can't resolve 'firebase/perf' in 'Project-Folder\node_modules\@angular\fire\performance\es2015'
```

**Comment by jamesdaniels** [comment]
FYI we'll be releasing 5.2 next week, feel free to try the beta at `@angular/fire@next` in the meantime.

**Comment by jamesdaniels** [comment]
Closing this since addressed, 5.2 release thread here #2086
